RepoDb.PostgreSql/RepoDb.PostgreSql/StatementBuilders/PostgreSqlStatementBuilder.cs

```diff
@@ -275,11 +275,9 @@ public PostgreSqlStatementBuilder()
                     var line = splitted[index].Trim();
 
                     // Set the return value
-                    var returnValue = identityField != null ?
-                        string.IsNullOrEmpty(databaseType) ?
-                            identityField.Name.AsQuoted(DbSetting) :
-                                string.Concat($"CAST({identityField.Name.AsQuoted(DbSetting)} AS {databaseType})") :
-                                    primaryField != null ? primaryField.Name.AsQuoted(DbSetting) : "NULL";
+                    var returnValue = string.IsNullOrEmpty(databaseType) ?
+                        identityField.Name.AsQuoted(DbSetting) :
```

Comment:
This may fail if the `identityField` is `null`, right? And why did we remove the logic of reusing the `primaryField`?

Reply:
Let me revalidate this line 266 `if (identityField != null)` logic, it seems wrong. I think, it should be removed. The `"NULL"` needs to be a part of the result atleast.

Reply:
want me to make that change on this PR?

Reply:
Yes please, since you had the latest and greatest copy.

Reply:
the diff is not handling it well. but it is done